app/gui/src/project-view/components/visualizations/TableVisualization/tableVizToolbar.ts

```diff
@@ -49,6 +49,7 @@ interface NewNodeOptions extends SortFilterNodesButtonOptions, ColumnNodeButton
 
 export interface RefreshButtonOptions {
   refreshGrid: () => void
+  isButtonDisabled: ToValue<boolean>
```

Comment:
This option of outside is called "isButton", but actually it is about button**s**. The fact that we want to block both buttons (new node and refresh) is a bit accidental. Either we should use different flags, or the name should be not about buttons (but something like "Table state changed").

Reply:
I'm thinking about this part: https://github.com/enso-org/enso/blob/develop/app/gui/src/project-view/components/visualizations/TableVisualization.vue#L1158

Any future programmer looking at this will assume the flag is about one button only, probably create node button judging by the value; there's no hint that more than one button will be affected by any change here.

Reply:
Better now?